shared/middleware/helpers/bucket_helper.rb

```diff
@@ -79,7 +79,11 @@ def list(encrypted_channel_id)
   end
 
   def get(encrypted_channel_id, filename, if_modified_since = nil, version = nil)
-    owner_id, channel_id = storage_decrypt_channel_id(encrypted_channel_id)
+    begin
+      owner_id, channel_id = storage_decrypt_channel_id(encrypted_channel_id)
+    rescue ArgumentError
```

Comment:
You should rescue from both `ArgumentError` and `OpenSSL::Cipher::CipherError`` here - different kinds of invalid input will cause the base64 decoder to raise one or the other. We consistently rescue from both in a bunch of places, such as https://github.com/code-dot-org/code-dot-org/blob/staging/shared/middleware/channels_api.rb#L46.